packages/lib-classifier/src/plugins/tasks/TextTask/models/TextPreviousAnnotations.js

```diff
@@ -0,0 +1,10 @@
+import { types } from 'mobx-state-tree'
+
+const TextPreviousAnnotations = types.model('TextPreviousAnnotations', {
```

Comment:
From the name, I expected this model to be a collection of `Annotation` models but I don't have any better suggestions for names either.

Reply:
How about `TextPreviousAnnotationValues`?

Reply:
Sounds good. I don't have a strong opinion on the name here.